src/io/platform/f3xx/CANf3xx.cpp

```diff
@@ -107,7 +107,7 @@ CAN::CANStatus CANf3xx::connect() {
     CAN_FilterTypeDef defaultFilter;
     defaultFilter.FilterIdHigh = 0;
     defaultFilter.FilterIdLow = 0;
-    defaultFilter.FilterMaskIdHigh = 0xFFFF;
+    defaultFilter.FilterMaskIdHigh = 0x0000;
```

Comment:
Comment on line 106 should reflect new changes if we are now accepting all messages.